test/fail_compilation/iasm1.d

```diff
@@ -49,6 +49,29 @@ void test2()
 
 /*********************************************/
 
+/* TEST_OUTPUT:
+---
+iasm1.d(306): Error: operand cannot have both R8 and [R9]
+iasm1.d(307): Error: operand cannot have both RDX and 0x3
```

Comment:
missing the "fail_compilation prefix/", to save you scrolling the log.

Reply:
Too late! But thanks for the help!